data/remnant missions.txt

```diff
@@ -493,5 +592,771 @@ mission "Remnant: Technology Available"
 				`	"I'm honored. Thank you."`
 					decline
 				`	"Is there anything else I can do to assist your people?"`
-			`	"Not that I am aware of right now," he says, "but we will call on you if a need ever arises."`
+			`	"There might be," he says. "Check the spaceports. If anyone has a job to be done, they will approach you. And there are almost always a few Korath ships that seem bent on destruction instead of loot, and those need to be dealt with."`
+				decline
+
+
+mission "Remnant: Heavy Laser"
+	name "Retrieve Heavy Lasers for the Remnant"
+	description "An engineer on <planet>, a Remnant world, has offered to pay you one million credits in exchange for obtaining two Heavy Lasers."
+	minor
+	source
+		government Remnant
+	to offer
+		has "license: Remnant"
+		random < 50
+	on offer
+		require "Heavy Laser"
+		conversation
+			`As you are browsing the spacesport shops, a woman approaches you and introduces herself as a researcher at one of their weapons facilities. She pauses before beginning to recite a quick tune: "We can see that your ship has weapons that match human techniques, but are unlike anything we have record of. What kind of weapons are they?"`
+			choice
+				`	"Heavy Lasers"`
+					goto HeavyLasers
+				`	"I'd rather not share that."`
+					decline
+			label HeavyLasers
+			`	"Really? Our records indicate that humanity had basic laser weapons when we left, but nothing that would be considered useful modern weapons." She pauses to look down at her scanner. "And yet our scans indicate that they are capable of significant power output."`
+			`	Based on these scans, we may well be able to learn quite a bit from these weapons. Would you be able to acquire a pair of them for us? We could offer you very good compensation for doing so."`
```

Comment:
I'm a little confused. Given this paragraph is a continuation of the same speaker, wouldn't I leave off the paranthese? Especially since I just deleted it off the end of the previous line?

Reply:
OK, resolved.